lib/reports/translation_word_report.rb

```diff
@@ -23,34 +23,33 @@ def self.generate_csv(start_date, end_date)
       raise ArgumentError, 'end_date cannot be earlier than the start date' if end_date < start_date
 
       CSV.generate do |csv|
-        locales = Project.pluck(:targeted_rfc5646_locales, :base_rfc5646_locale).map { |hash, base| hash.keys - [base]}.flatten.uniq
+        locales = locales = Project.pluck(:targeted_rfc5646_locales, :base_rfc5646_locale).map { |hash, base| hash.keys - [base]}.flatten.map!(&:downcase).uniq.sort
```

Comment:
`locales` is being assigned twice here and `map!` is not needed when the result is being saved in a var. Only problem I see so I will go ahead and merge and fix in a followup commit.

Reply:
Sorry about that, copy and paste error from the Rails Console. Thanks for fixing!